### ISSUE TITLE
Locks

### ISSUE THREAD

**Issue by AlfonzDivoch** [MaDHouSe79/mh-vehiclekeys#4]
When i type in the config 
Config.LockNPCDrivingCars = false -- Lock state for NPC cars being driven by NPCs [true = locked, false = unlocked]
Config.LockNPCParkedCars = false -- Lock state for NPC parked cars [true = locked, false = unlocked]

Its still locked

**Comment by MaDHouSe79** [comment]
download a new one and see if it work.

**Comment by AlfonzDivoch** [comment]
Yes that works now, but could you make it that when you drag out an npc the npc would give you keys? Because when i drag npc out i still need to press H to find them

**Comment by MaDHouSe79** [comment]
try again i change the code, but you need to change Config.UseHotwiring = true to false in de config.lua
